StartHELayers.sh

```diff
@@ -0,0 +1,205 @@
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+
+# This script is intended for those who want to try a pre-built docker container for their toolkit distribution of choice.
+# see the script called BuildDockerContainer if you want to build or modify your own instance. 
+
+source ConfigConstants.sh
+
+bold=$(tput bold)
+normal=$(tput sgr0)
+
+BASEDIR="$PWD"/$(dirname $0)
+SCRIPTNAME=$(basename $0)
+
+
+print_usage(){
+  cat <<EOF
+
+  ${bold}Usage: $SCRIPTNAME [options] LAB_TYPE${normal}
+
+  ${bold}LAB_TYPE${normal}   Selects the type of lab to
+                 download the HELayers container from Docker Hub.
+                 Available lab types are:
+                 x86_64/amd64: {lab, python}
+                 s390x:        {lab, python}
+
+  ${bold}OPTIONS:${normal}
+  -h             Displays this help information.
+
+  ${bold}Examples:${normal}
+  Obtain the HELayers container with the python lab.
+  $SCRIPTNAME python
+
+  Show this help message.
+  $SCRIPTNAME -h
+
+
+EOF
+}
+
+# The number of parameters passed to this script
+NPARAM=$#
+
+# First we determine which architecture we are running on... AMD64 or s390x
+ARCH=`uname -m`
+
+if [[ "$ARCH" == "x86_64" ]] || [[ "$ARCH" == "amd64" ]]; then
+  #echo "Determined AMD64/x86_64 Architecture"
+  architecture="AMD64/x86_64"
+  ARCH="amd64"
+elif [[ $ARCH == "s390x" ]]; then
+  # echo "Determined s390x Architecture"
+  architecture="s390x"
+else
+  echo " "
+  echo " FATAL: Aborting. HELayers is not supported on the $ARCH platform."
+  echo " "
+  exit -1
+fi
+
+#echo "$@"
+NPARAM=$#
+
+while getopts ":h" opt; do
+  case ${opt} in
+    h ) # Usage
+      print_usage
+      exit 0
+      ;;
+   \? ) # Usage
+     echo "Fatal: Invalid options"
+     print_usage
+     exit -2
+     ;;
+   : ) # Invalid option. Print usage
+     echo "Fatal: Invalid option: $OPTARG requires an argument" 1>&2
+     print_usage
+     exit -3
+  esac
+done
+
+shift $((OPTIND -1))
+#echo "$@"
+
+OPTINDNUMBER=$((OPTIND-1))
+
+if [[ $((NPARAM - OPTINDNUMBER)) -ne 1 ]]; then
+  echo "Fatal: Options and parameters mismatch or missing parameters. Please check order and number of parameters."
+  print_usage
+  exit -4
+fi
+
+labtype=$1
+if [ $ARCH == "s390x" ]
+then
+  if [[ "$labtype" = "python" ]]
+  then
+      imagesuffix="pylab-s390x"
+  elif [[ "$labtype" = "lab" ]]
+  then
+      imagesuffix="lab-s390x"
+  else
+    echo "INFO: Lab type: $labtype on $ARCH is not supported on this release."
+    echo "      Please choose a supported lab type for your trials by specifying"
+    echo "      the command: ${bold}$SCRIPTNAME <lab | python>${normal}"
+    echo " "
+    exit -1
+  fi
+else
+  if [ $ARCH == "amd64" ]; then
+    if [[ "$labtype" = "python" ]]
+      then
+      imagesuffix="pylab"
+      labport="8888"
+    elif [[ "$labtype" = "lab" ]]
```

Comment:
thinking the other option should be `cpp` instead of lab.  Lab is a little ambiguous, unless you have a better idea

Reply:
sure, I can change `lab` to `cpp`, that makes sense to me.